### PR TITLE
docs: add deprecation notice

### PR DESCRIPTION
**Pull request by Damego**
## Pull Request Type

- [ ] Feature addition
- [ ] Bugfix
- [x] Documentation update
- [ ] Code refactor
- [ ] Tests improvement
- [ ] CI/CD pipeline enhancement
- [ ] Other: [Replace with a description]

## Description

im bad at explaining stuff

## Python Compatibility

- [ ] I've ensured my code works on Python `3.10.x`
- [ ] I've ensured my code works on Python `3.11.x`

## Checklist

- [ ] I've run the `pre-commit` code linter over all edited files
- [ ] I've tested my changes on supported Python versions
- [ ] I've added tests for my code, if applicable
- [ ] I've updated / added  documentation, where applicable

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ interactions.py
 
 ----
 
+Deprecation Notice
+******************
+Please note, this version is now deprecated. We are recommending migrate to v5. `Migration guide from v4`_
```
Comment (mAxYoLo01) [inline README.rst]:
```suggestion
Please note, this version is now deprecated. We are recommending you migrate to v5. `Migration guide from v4`_
```

File: docs/index.rst [hunk 2]
```diff
@@ -1,5 +1,9 @@
 .. image:: _static/banner.png
 
+Deprecation Notice
+=================
+Please note, this version is now deprecated. We are recommending migrate to v5. `Migration guide from v4`_
```
Comment (mAxYoLo01) [inline docs/index.rst]:
```suggestion
Please note, this version is now deprecated. We are recommending you migrate to v5. `Migration guide from v4`_
```

**Comment by Damego** [comment]
pre commit, isn't comment `isort: skip` a joke for you?

**Comment by Damego** [comment]
Moved to the fork

**Pull request closed without merging**